test/libgit2.jl

```diff
@@ -340,6 +340,11 @@ mktempdir() do dir
     #@testset "Fetch from cache repository" begin
         repo = LibGit2.GitRepo(test_repo)
         try
+            # Set the username and email for the test_repo
+            cfg = LibGit2.GitConfig(repo)
+            LibGit2.set!(cfg, "user.name", "AAAA")
+            LibGit2.set!(cfg, "user.email", "BBBB@BBBB.COM")
```

Comment:
would it maybe make sense to wait to set this until as late as possible, only for the part of the test that needs it?